edx_sga/sga.py

```diff
@@ -491,6 +495,16 @@ def download_zip(self, student_files):
         # save zip file and return its URL as JSON response
         return Response(json={'zip_url': default_storage.url(default_storage.save(zip_filename, sio))})
 
+    def get_submission_download_status(self, student_id):
+        student_item_dict = self.student_submission_id(student_id)
```

Comment:
You don't need this line since you're not doing anything with this variable except passing it to the other method.
You can simply call `student_submission_id` as an argument.